scapy/layers/inet.py

```diff
@@ -261,8 +261,8 @@ class IPOption_SDBM(IPOption):
      28: ("UTO", "!H"),
      34: ("TFO", "!II"),
      # RFC 3692
-     253: ("Experiment", "!HHHH"),
-     254: ("Experiment", "!HHHH"),
+     # 253: ("Experiment", "!HHHH"),
```

Comment:
Is there a reason for this?

Reply:
Just below is another array with the reversed keys (name -> id), which does not contain the experimental values. This means that adding an option `("Experiment", b"data")` fails to resolve.

When building the random options, if the value is picked up, it will fail.